src/components/SecondaryPanes/Breakpoints/ExceptionOption.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+import React from "react";
+
+type ExceptionOptionProps = {
+  className: string,
+  isChecked: boolean,
+  label: string,
+  onChange: Function,
+  shouldRender: boolean
```

Comment:
Instead of shouldRender, IMO we should just avoid putting the component in the parent's JSX.  Is there any reason to render it at all if the first checkbox isn't checked?